### PR TITLE
Fix issue #963 - Khmer Translation on Law Page

### PR DESCRIPTION
**Pull request by S-mardii**
https://github.com/OpenDevelopmentMekong/opendev-wptheme/issues/963

### CODE REVIEW

File: single.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
     $local_lang = '_'.$languages[1];
 } else {
     $lang = '';
+    echo "ttit";
```
Comment (acorbi) [inline single.php]: @S-mardii @Huyeng is this intended? or is it a debug message that you forgot to remove?

**Comment by S-mardii** [comment]
@acorbi  that line is just my first testing on when I start to use the fork.

File: single.php (hunk 1 above)
Comment (S-mardii) [inline single.php]: @acorbi It was my first testing when I start using the Fork account. I will take a look at that again

Comment (S-mardii) [inline single.php]: @acorbi  and that is just my debug message which i forgot to remove.

Comment (acorbi) [inline single.php]: Please remove that debug message and push the changes so the PR gets updated or open a new one if needed

**Comment by acorbi** [comment]
@S-mardii I saw you have created another PR on https://github.com/OpenDevelopmentMekong/opendev-wptheme/pull/968 which replaces this one. If so, please close this one since it is not valid anymore...

**Pull request closed without merging**